packages/salesforcedx-utils/jest.config.js

```diff
@@ -0,0 +1,8 @@
+const baseConfig =  require('../../config/jest.base.config');
+
+module.exports = Object.assign({},
```

Comment:
packages/salesforcedx-apex-debugger/jest.config.js matches packages/salesforcedx-apex-debugger/jest.integration.config.js, but they don't match packages/salesforcedx-utils/jest.config.js.  Should they all match & be the same?

Reply:
Not necessarily.  The reason a config file exists in every package is so that it can have individual jest configs if necessary. (which I started I didn't know if all the files would be consistent).  Integration & unit config should def differ in that those use a different base config.  I added the configs 1 by 1 as I went so I could have changed style after a bit.  I'll review the files and ensure they look consistent.